lib/schema.js

```diff
@@ -4,15 +4,32 @@ const Joi = require('joi');
 
 const internals = {};
 
+// internals.model = Joi.func().keys({
+//     tableName: Joi.string().required()
+// }).unknown();
 internals.model = Joi.func();
 
 exports.plugin = Joi.object({
+
+    knexFile: Joi.object(),
+    migration: Joi.object({
```

Comment:
I'm a little confused, so you're saying we should be able to pass in a migrations config per server registration and per plugin?